operations/helm/charts/mimir-distributed/values.yaml

```
@@ -350,6 +353,25 @@ alertmanager:
       cpu: 10m
       memory: 32Mi
 
+  # -- Fallback config for alertmanager
+  # When a tenant doesn’t have a Alertmanager configuration, the Grafana Mimir Alertmanager uses the fallback configuration.
+  fallbackConfig: |
+    global:
+        smtp_smarthost: 'localhost:25'
+        smtp_from: 'alertmanager@example.org'
+        smtp_auth_username: 'alertmanager'
+        smtp_auth_password: 'password'
+    route:
+        receiver: 'default-receiver'
+        group_wait: 30s
+        group_interval: 5m
+        repeat_interval: 4h
+        group_by: [cluster, alertname]
+    receivers:
+        - name: 'default-receiver'
+          email_configs:
+          - to: 'alerts@example.org'
```

@krajorama suggested a more simplistic config that doesn't send emails

```suggestion
    route:
      receiver: "example_receiver"
      group_by: ["example_groupby"]
    receivers:
      - name: "example_receiver"
```

---

I think this got lost during some rebase